crates/stc_ts_file_analyzer/src/analyzer/enums.rs

```diff
@@ -655,3 +671,14 @@ impl Visit<RExpr> for LitValidator<'_> {
         }
     }
 }
+
+fn has_not_str_and_not_plus(bin: &RBinExpr) -> bool {
```

Comment:
Can you rename this to `non_str_nor_plus`?

Reply:
fixed it